pymc3/distributions/multivariate.py

```diff
@@ -113,7 +124,10 @@ def logp(self, value):
         delta = value - mu
         k = tau.shape[0]
 
-        result = k * tt.log(2 * np.pi) + tt.log(1. / det(tau))
+        if self.gpu_compat:
+            result = k * tt.log(2 * np.pi) + tt.log(1. / det(tau))
```

Comment:
You can use [this code](https://github.com/pymc-devs/pymc3/pull/1694/files#diff-e87226e0c9d21354a0fdf3a0e166ea18R161)  in future, it worked for me